### ISSUE TITLE
"Hidden links" toolbar icon not obvious

### ISSUE THREAD

**Issue by tuomas2** [AndBible/and-bible#3152]
Due to #3064 there is added new strongs mode "Hidden Links". That mode is not obvious from the icon. Icon should be changed. 

Is there any reason why this new "Hidden Links" mode could not replace "Off" mode?

@agrogers 

https://support.andbible.org/scp/tickets.php?id=2340

> I have noticed that the Hebrew & Greek button has an extra click on it, but I don't understand what it does. It's #2 here below:
> 
> 1. Bold font on button
> 2. Regular font on button
> 3. Underline words in text
> 4. Strong's numbers in text.

**Comment by agrogers** [comment]
At present the strongs button takes up a lot of space in the little action
dialog. I would like to make an option so that that button becomes one of
the icon buttons in the top row... then it wouldnt take up any space.

While it takes up space in the dialog, I wouldn't remove the "Off" mode. I
turn it off when I want that realestate.

As for a different icon, I am not sure what. I think changing the "Off"
icon is probably my preference - maybe it has a cross through it or
something. This still wont explain what the normal icon does.

___________________________
*www.TenOutOfTen.org* <https://www.TenOutOfTen.org>
***@***.***
(+61) 0477 174 193 (Australia)
(+95) 09 250018669 (Myanmar)

**Comment by agrogers** [comment]
This is a good example of why we need #2448. The potential inconvenience in that for a few people is trivial but the benefits to the majority is significant.

**Comment by tuomas2** [comment]
I disagree about #2448 because I think it's not necessary to try to push the knowledge about every small feature to every user (only those people that use strongs need that knowledge in the first place, not all users), but agree that we should have a help popup dialog ((?)-button) in strongs preferences screen that explains the non-obvious feature.

**Comment by tuomas2** [comment]
I'm thinking that we could should probably remove "Hidden Links" mode and change "Off" mode to act like this mode. It is true that it adds that 1 extra button but reduced confusion of a mode that is not self-explanatory is greater good I believe.

**Comment by agrogers** [comment]
Yeah, if there is no performance difference with the Strongs on or off then
off could be removed.

___________________________
*www.TenOutOfTen.org* <https://www.TenOutOfTen.org>
***@***.***
(+61) 0477 174 193 (Australia)
(+95) 09 250018669 (Myanmar)

**Comment by timbze** [comment]
Maybe this should be a new feature request but I'd also like to suggest that underline mode opens dictionary when clicking a word even if there's a bookmark on the verse. Now it will open the actions popup if there's a bookmark on the verse. When I enable underline mode my intention is word lookup and don't want to see bookmarks.

**Comment by tuomas2** [comment]
Another user confused about "Hidden links" feature: https://support.andbible.org/scp/tickets.php?id=2714

**Comment by tuomas2** [comment]
Took me a while to figure out the issue with the user. I feel this need to be changed.

**Comment by agrogers** [comment]
Onboarding or a Tips system would be a step towards solving this problem
and others like it. The first time they use the 'Hidden links' option it
pops up a message explaining what it does.

 From reading the Play Store reviews (which are overwhelmingly positive -
well done!), there are definitely users who don't stick with AB because
they can't work simple things (like selecting a verse!) in the first 5
mins. I think many features go unused because people don't know about
them... or think they are bugs :)

___________________________
*www.TenOutOfTen.org* <https://www.TenOutOfTen.org>
***@***.***
(+61) 0477 174 193 (Australia)
(+95) 09 250018669 (Myanmar)

**Comment by tuomas2** [comment]
I'll leave the option for now but removing it from MainBibleActivity's strongs button cycle. Now it has 3 modes, but "Hidden links" can be found from Text Options still.

**Comment by agrogers** [comment]
>>> but "Hidden links" can be found from Text Options still.

Is that correct? Do you mean long press the language button at the top. I
had forgotten that option was there. I thought the hidden links had been
removed... nice to find it again.
___________________________
*www.TenOutOfTen.org* <https://www.TenOutOfTen.org>
***@***.***
(+61) 0477 174 193 (Australia)
(+95) 09 250018669 (Myanmar)
